schemas/aws/account-1.yml

```diff
@@ -103,6 +103,12 @@ properties:
     enum:
     - aws
     - aws-us-gov
+  terraformState:
+    type: array
```

Comment:
why is this an array?

Reply:
We want to be able to reference the terraform resource schema multiple times because there will be multiple integrations for the same bucket